charts/flyte-core/templates/clusterresourcesync/cronjob.yaml

```diff
@@ -10,10 +10,11 @@ spec:
   jobTemplate:
     spec:
       template:
-        {{- with .Values.flyteadmin.podAnnotations }}
         metadata:
-          annotations: {{ tpl (toYaml .) $ | nindent 12 }}
-        {{- end }}
+          annotations:
+            {{- with .Values.flyteadmin.podAnnotations }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
         spec:
```

Comment:
small change, Add annotations if they are passed or else don't specify them in generated template.
```yaml
        metadata:
          {{- with .Values.flyteadmin.podAnnotations }}
          annotations:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```

Reply:
I've updated this 👍